handlers/handlers.go

```diff
@@ -261,12 +262,10 @@ func readDataAggregation(w http.ResponseWriter, req *http.Request, cfg *config.C
 		return
 	}
 	validatedQueryParams, validationErrs := data.ReviewDataAggregationQuery(ctx, cfg, urlQuery, censusTopicCache)
-	for _, vErr := range validationErrs {
-		if vErr.ID != DateFromErr && vErr.ID != DateToErr {
-			log.Error(ctx, "unable to review query", errors.New(vErr.Description.Text))
-			setStatusCode(w, req, errors.New(vErr.Description.Text))
-			return
-		}
+	if len(validationErrs) > 0 {
+		m := mapper.CreateDataAggregationPage(cfg, req, rend.NewBasePageModel(), validatedQueryParams, []data.Category{}, []data.Topic{}, &searchModels.SearchResponse{}, lang, zebedeeCli.HomepageContent{}, "", navigationCache, template, topicModels.Topic{}, validationErrs)
+		buildDataAggregationPage(w, m, rend, template)
+		return
 	}
```

Comment:
- I think a little comment on what the review does what you're doing with the errors would be helpful.
- Logging the errors at some point might also not be a bad idea.